packages/kit/src/index.tsx

```diff
@@ -12,6 +12,5 @@ export const RiseComponents = {
   },
   LucideIcon: {
     component: LucideIcon,
-    validator: LucideIcon.validate,
```

Comment:
🥲

Reply:
Haha yeah, I had to let this one go for now. We need better validator for Icons for this and other Tamagui props (which is generally tricky, but we have it on the roadmap!)